cmake/coverallsGcovJsons.cmake

```diff
@@ -110,14 +110,15 @@ endmacro()
 
 # Get the coverage data.
 file(GLOB_RECURSE GCDA_FILES "${COV_PATH}" "*.gcda")
-message("GCDA files:")
+message("Process GCDA files:")
+message("===============================")
 
 # Get a list of all the object directories needed by gcov
 # (The directories the .gcda files and .o files are found in)
 # and run gcov on those.
 foreach(GCDA ${GCDA_FILES})
-	message("Process: ${GCDA}")
-	message("------------------------------------------------------------------------------")
+	# message("Process: ${GCDA}")
+	# message("------------------------------------------------------------------------------")
```

Comment:
Either remove or uncomment it.

Do not comment lines in code under version control system(git). Because git will track file changes through history and record every commit forever.